tester_app/app/assets/javascripts/application.js

```diff
@@ -1,3 +1,17 @@
+
+// It is important that generated/vendor-bundle must be before bootstrap since it is
+// exposing jQuery and jQuery-ujs.
+// DO NOT REQUIRE jQuery or jQuery-ujs in this file!
+// DO NOT REQUIRE TREE!
+// Must be on top
+//= require react_on_rails
```

Comment:
I think react_on_rails can be later, right? or should it go earlier? I don't think there's any connection as it does not do any monkey patching